.github/workflows/CI.yml

```diff
@@ -41,9 +46,9 @@ jobs:
       - uses: actions/checkout@v4
       - name: Build and Run Tests
         run: |
-          ./gradlew check
+          ./gradlew check -x integTest -x yamlRestTest -x spotlessJava
```

Comment:
We need to make run the complete gradle check suite temporary. Ref: https://github.com/opensearch-project/flow-framework/pull/169/files

Reply:
I see, I'll go ahead and omit this modification to the CI.yml